src/components/DeleteRoom.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Card, Button, Typography } from '@material-tailwind/react';
+import { fetchRooms, destroyRoom } from '../redux/features/rooms/roomsSlice';
+
+const DeleteRoom = () => {
```

Comment:
- [ ] In the mobile view, in the delete section, there is no hamburger menu.

![image](https://github.com/ProgramKingAbel/hotel-hub-frontend/assets/57421823/7de45a6f-59ef-4d3c-8376-8f229e3c763f)


This means there is no way to get to the main section of the app, apart from using the "back" button in the browser.

Kindly, provide a way to go back either by showing the hamburger menu, or with a "back" button. 👍